docker/upgrade.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:24.04
+
+ARG TARGETARCH
+
+RUN apt-get update \
+    &&  apt-get install -y curl libcurl4 wait-for-it tini \
+    nodejs  \
+    npm \
+    && npm install -g ts-node typescript \
+    && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y \
```

Comment:
I think we don't need a rust toolchain in this container, but maybe I'm missing something.

Reply:
that's right, i was trying some rust tests and forgot to remove it https://github.com/EspressoSystems/espresso-network/commit/f0a148fe64314e44205e5a21be00aa9fbbcfa06a